integration/test_orgs.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+
+import pytest
+
+timestamp = str(int(datetime.utcnow().timestamp()))
+new_org = "integration test org {}".format(timestamp)
+
+
+@pytest.fixture(scope="module")
+def org(connection):
+    orgs_gen = connection.orgs.get_all()
+    orgs = next(orgs_gen)
+    # Assumption: Parent org always exists.
```

Comment:
Even the parent org has a `parent_org_id` and `parent_org_uid`, which one would assume it won't exist for the first org, for some reason they do and hence had to make the assumption.